app/src/test/java/org/mozilla/fenix/settings/creditcards/StringTest.kt

```diff
@@ -42,15 +42,13 @@ class StringTest {
         val jcbCard = "3530111333300000"
         val masterCardCard = "5555555555554444"
         val visaCard = "4111111111111111"
-        val voyagerCard = "869941728035895"
 
         assertTrue(americanExpressCard.validateCreditCardNumber())
         assertTrue(dinnersClubCard.validateCreditCardNumber())
         assertTrue(discoverCard.validateCreditCardNumber())
         assertTrue(jcbCard.validateCreditCardNumber())
         assertTrue(masterCardCard.validateCreditCardNumber())
         assertTrue(visaCard.validateCreditCardNumber())
-        assertTrue(voyagerCard.validateCreditCardNumber())
```

Comment:
Why did we remove this assertion?

Reply:
We don't support voyager in our recognized network so this will always fail.